node/src/neighborhood/mod.rs

```diff
@@ -3978,29 +4033,39 @@ mod tests {
         assert_eq!(subject.overall_connection_status.can_make_routes(), true);
         assert_eq!(
             subject.overall_connection_status.stage(),
-            OverallConnectionStage::ThreeHopsRouteFound
+            OverallConnectionStage::RouteFound
         );
         assert_eq!(
             node_to_ui_message,
             &NodeToUiMessage {
                 target: MessageTarget::AllClients,
                 body: UiConnectionChangeBroadcast {
-                    stage: UiConnectionStage::ThreeHopsRouteFound
+                    stage: UiConnectionStage::RouteFound
                 }
                 .tmb(0),
             }
         );
         TestLogHandler::new().exists_log_containing(&format!(
-            "DEBUG: {}: The connectivity check has found a 3 hops route.",
-            test_name
+            "DEBUG: {}: The connectivity check has found a {}-hop(s) route.",
```

Comment:
Regardless of the value of the number, "-hop" is perfectly valid English. 0-hop, 1-hop, 100000-hop, all of that is fine. So you could get rid of the `(s)` here, if you wanted.